guidelines/index.html

```diff
@@ -78,1254 +106,1196 @@ <h3>Section status levels</h3>
-				</ul>
-			</div>
 
-			<section>
-			<h3>Outcomes</h3>
-			<p>Outcomes are verifiable statements that allow testers to reliably determine if the content being evaluated satisfies the user needs identified in the Guideline. All outcomes and assertions that relate to a Guideline will be listed together to encourage adoption of higher levels of accessibility. </p>
-			<p>Each outcome is associated with at least one <a>method</a>. Methods are informative and kept in how to documents. Each method contains techniques for meeting the outcome, examples, resources, and <a>sets of tests</a>  for evaluating the outcome. Methods can apply to a specific technology, such as HTML, or can be more generic where the advice applies no matter what technology, such as the methods supporting the Clear Language guideline. </p>
+				<section>
+					<h4>Control information</h4>
+					<p class="guideline-text">Users have information about interactive components that is identifiable and usable visually and using assistive technology.</p>
 
-			<p>Outcomes are written so that testers can determine the accessibility of technologies based solely on the outcome, even when methods do not yet exist for those technologies.</p>
+					<section class="provision" data-status="exploratory">
+						<h5>Control importance</h5>
+						<p class="provision-text">The importance of interactive components is indicated.</p>
+						<p class="ednote">Needs <a href="#additional_research">additional research</a></p>
+					</section>
+					<section class="provision" data-status="exploratory">
+						<h5>Control labels</h5>
+						<p class="provision-text">Interactive components have visible labels that identify the purpose of the component.</p>
+					</section>
+					<section class="provision" data-status="exploratory">
+						<h5>Control updates</h5>
+						<p class="provision-text">Changes to interactive components’ names, roles, values or states are visually and programmatically indicated.</p>
+					</section>
+					<section class="provision" data-status="exploratory">
+						<h5>Distinguishable controls</h5>
+						<p class="provision-text"> Interactive components are visually distinguishable without interaction from static content and include visual cues on how to use them.</p>
+					</section>
+					<section class="provision" data-status="exploratory">
+						<h5>Field constraints</h5>
+						<p class="provision-text">Field constraints and conditions (required line length, date format, password format, etc) are available.</p>
+					</section>
+					<section class="provision" data-status="exploratory">
+						<h5>Input labels</h5>
+						<p class="provision-text">Inputs have visible labels that identify the purpose of the input.</p>
+					</section>
+					<section class="provision" data-status="exploratory">
+						<h5>Label in name</h5>
+						<p class="provision-text">The programmatic name includes the visual label.</p>
```

Comment:
I feel something like this could make it clearer what to include:

```suggestion
						<p class="provision-text">The programmatic name includes all of the text that is visibly part of the label.</p>
```

Reply:
Clearer, but potentially a different scope. I'd rather leave this to the sub-group to work on.